content/blog/2019-02-18-stats19.md

```diff
@@ -0,0 +1,1156 @@
+---
+slug: "stats19"
+title: "stats19: a package for road safety research"
+package_version: 0.2.0
+authors:
+  - Layik Hama
+  - Robin Lovelace
+date: 2019-2-18
```

Comment:
```suggestion
date: 2019-02-18
```

Reply:
Thanks @stefaniebutland. I had tried that too and just tried it, on my dev env still shows `January 1, 0001`

Reply:
@layik I've made a suggested commit that should fix the date issue.
I will review the post today.

Reply:
Thank you.